op-program/client/program.go

```diff
@@ -46,23 +46,12 @@ func RunProgram(logger log.Logger, preimageOracle io.ReadWriter, preimageHinter
 	l1PreimageOracle := l1.NewCachingOracle(l1.NewPreimageOracle(pClient, hClient))
 	l2PreimageOracle := l2.NewCachingOracle(l2.NewPreimageOracle(pClient, hClient))
 
-	bootInfo := NewBootstrapClient(pClient).BootInfo()
-	logger.Info("Program Bootstrapped", "bootInfo", bootInfo)
-	safeHead, outputRoot, err := tasks.RunDerivation(
-		logger,
-		bootInfo.RollupConfig,
-		bootInfo.L2ChainConfig,
-		bootInfo.L1Head,
-		bootInfo.L2OutputRoot,
-		bootInfo.L2ClaimBlockNumber,
-		l1PreimageOracle,
-		l2PreimageOracle,
-	)
-	if err != nil {
-		return err
+	bootInfo := boot.NewBootstrapClient(pClient).BootInfo()
+	if os.Getenv("USE_INTEROP") == "true" {
```

Comment:
nit: Worth renaming this to `OP_PROGRAM_CLIENT_USE_INTEROP` just to be super-super clear that this is a client-only envar. If the host were to accidentally be run with `USE_INTEROP=true`, then the execution fork bombs.

Reply:
Good idea. Done.